js/sunburst.d3.js

```diff
@@ -90,7 +90,7 @@ function Sunburst(data, { // data is either tabular (array of objects) or hierar
   
     cell.append("path")
         .attr("d", d => arc(d.current))
-        .attr("fill", d => color ? color(d.ancestors().reverse()[1]?.index) : fill)
+        .attr("fill", d => (d.data.disabled || d.data.disabled === undefined && d.parent) ? "#000" : color ? color(d.ancestors().reverse()[1]?.index) : fill)
```

Comment:
I want to understand what's going on here, because it looks cool and I don't get it :) Is it chaining if else statements somehow? It's the second `?` that confuses me.

Reply:
It's indeed chaining two if/else statements. It's not very legible and therefor pretty bad. It also mixes visuals with data pretty badly, I'll fix it at some point when moving to Vue or similar.